coremis_app_integration/endpoints/change_password.py

```diff
@@ -0,0 +1,65 @@
+import re
+
+from dataclasses import asdict
+from django.utils.translation import gettext_lazy as _
+from rest_framework.authentication import authenticate
+from rest_framework.decorators import (
+    authentication_classes,
+    permission_classes,
+    api_view
+)
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import status
+
+from coremis_app_integration.authorization import CustomBasicAuthentication
+from coremis_app_integration.dataclasses import (
+    FailedResponse,
+    SuccessResponse
+)
+
+
+REGEX_PATTERN = r'^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&!+=])(?=\S+$).{8,}$'
```

Comment:
Please add some comment about the requirement coming from this regex

```suggestion
"""
In summary, the regex enforces a string (likely a password) to:
    Be at least 8 characters long
    Contain at least one digit
    Contain at least one lowercase letter
    Contain at least one uppercase letter
    Contain at least one special character from the set @#$%^&!+=
    Not contain any whitespace characters
"""

REGEX_PATTERN = r'^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&!+=])(?=\S+$).{8,}$'
```